### DISCUSSION TITLE
About security

### DISCUSSION THREAD

**Discussion by claudiojulioferraz** [standardnotes/forum#2715] [General] [answered]
How are notes encrypted to the database if autosync and autosave are running all the time? Are notes encrypted with each character sent?

_Automatically translated_

**Comment by JaspalSuri** [comment, marked as answer]
Hi @claudiojulioferraz,

The notes are encrypted before they are (1) written your device's disk and (2) sent to our servers (if you're signed in) as the decrypted content is stored in your computer's memory (such as your RAM) before being deleted.

For more information, please [see here](https://blog.standardnotes.org/enhanced-security-with-device-storage-encryption/).

**Comment by claudiojulioferraz** [comment]
Thanks!

**Comment by JaspalSuri** [comment]
You're welcome! Feel free to reach out to us with any other questions in the future!
